Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1011,9 +1011,13 @@ void MenuBar::UpdateToolsMenu(bool emulation_started)
 
     const QString sysmenu_version =
         tmd.IsValid() ?
-            QString::fromStdString(DiscIO::GetSysMenuVersionString(tmd.GetTitleVersion())) :
+            QString::fromStdString(DiscIO::GetSysMenuVersionString(tmd.GetTitleVersion(), tmd.IsvWii())) :
             QString{};
-    m_boot_sysmenu->setText(tr("Load Wii System Menu %1").arg(sysmenu_version));
+
+    const char* sysmenu_text = (tmd.IsValid() && tmd.IsvWii()) ?
+        "Load vWii System Menu %1" : "Load Wii System Menu %1";
```

Comment:
This breaks translations. The string literal must be inside the `tr(` in the source code, otherwise the script that picks up translatable strings won't find it.

Reply:
Would this be fine for the script?
```c++
    const QString sysmenu_text = (tmd.IsValid() && tmd.IsvWii()) ?
        tr("Load vWii System Menu %1") : tr("Load Wii System Menu %1");

    m_boot_sysmenu->setText(sysmenu_text.arg(sysmenu_version));
```

Reply:
Yep, that's good.